src/cephadm/cephadm

```diff
@@ -759,6 +759,28 @@ def infer_fsid(func):
 
     return _infer_fsid
 
+def get_last_local_ceph_image():
+    """
+    :return: The most recent local ceph image (already pulled)
+    """
+    out, _, _ = call_throws(
+        [container_path, 'images', '--format', '{{ .ID }} {{.Repository}} {{.Tag}}'])
```

Comment:
both docker and podman return images in age sorted order.. so i think we can just use the first image that has the right tags?

Reply:
Done